src/index.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (C) 2019 The "mysteriumnetwork/js-tequilapi" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import TequilapiClientFactory, { TEQUILAPI_URL } from './tequilapi-client-factory'
+
+export default TequilapiClientFactory
+export { TEQUILAPI_URL }
```

Comment:
I dont think it's a good idea to export constant `TEQUILAPI_URL`, is caller is not enough with provider factory, he implements his own factory

Reply:
Not sure I get that - you mean that if user implements his own factory, than he has to have `'http://127.0.0.1:4050'` constant in his own project instead of using `TEQUILAPI_URL`?